cocos2d/CCDirector.m

```diff
@@ -970,16 +968,17 @@ -(void) createStatsLabel
 	NSData *nsdata = [NSData dataWithBytes:data length:data_len];
 	CGDataProviderRef imgDataProvider = CGDataProviderCreateWithCFData( (__bridge CFDataRef) nsdata);
 	CGImageRef imageRef = CGImageCreateWithPNGDataProvider(imgDataProvider, NULL, true, kCGRenderingIntentDefault);
-	CCTexture *texture = [[CCTexture alloc] initWithCGImage:imageRef contentScale:contentScale];
 	CGDataProviderRelease(imgDataProvider);
+    
+    CCImage *image = [[CCImage alloc] initWithCGImage:imageRef contentScale:contentScale options:nil];
```

Comment:
Aha, found a different indentation level! It's easier to find them in git than xcode.